### PR TITLE
[INLONG-6786][Dashboard] Supoort Apache Hudi sink management

### LINKED ISSUE #6786: [Feature][Dashboard] Supoort Apache Hudi sink management
### Description

[Dashboard] Supoort Apache hudi in frontend

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ZuoFengZhang**
### Prepare a Pull Request
*[Dashboard] Supoort Apache hudi in frontend*

- Fixes #6786 

### Motivation

image

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by healchow — APPROVED** [review]
LGTM

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -177,6 +177,25 @@
...
+  "meta.Sinks.Hudi.Description": "表描述",
+  "meta.Sinks.Hudi.ExtList": "属性",
+  "meta.Sinks.Hudi.DataConsistency": "数据一致性",
+  "meta.Sinks.Hudi.FieldName": "字段名",
+  "meta.Sinks.Hudi.FieldNameRule": "以英文字母或下划线开头，只能包含英文字母、数字、下划线",
+  "meta.Sinks.Hudi.FieldType": "字段类型",
+  "meta.Sinks.Hudi.FieldDescription": "字段描述",
+  "meta.Sinks.Hudi.IsPrimaryKey": "主键",
+  "meta.Sinks.Hudi.PrimaryKey": "主键",
+  "meta.Sinks.Hudi.IsPartitionKey": "分区字段",
```
Comment (leezng) [inline inlong-dashboard/src/locales/cn.json:193]: Is this key(`IsPartitionKey`) not used?

File: inlong-dashboard/src/locales/cn.json [hunk 2]
```diff
@@ -177,6 +177,25 @@
...
+  "meta.Sinks.Hudi.Warehouse": "仓库路径",
+  "meta.Sinks.Hudi.FileFormat": "⽂件格式",
+  "meta.Sinks.Hudi.Description": "表描述",
+  "meta.Sinks.Hudi.ExtList": "属性",
+  "meta.Sinks.Hudi.DataConsistency": "数据一致性",
+  "meta.Sinks.Hudi.FieldName": "字段名",
+  "meta.Sinks.Hudi.FieldNameRule": "以英文字母或下划线开头，只能包含英文字母、数字、下划线",
+  "meta.Sinks.Hudi.FieldType": "字段类型",
+  "meta.Sinks.Hudi.FieldDescription": "字段描述",
+  "meta.Sinks.Hudi.IsPrimaryKey": "主键",
```
Comment (leezng) [inline inlong-dashboard/src/locales/cn.json:191]: Is this key(`IsPrimaryKey`) not used?

**Review by leosanqing — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json (hunk 1 above)
Comment (ZuoFengZhang) [reply inlong-dashboard/src/locales/cn.json:193]: done

**Review by dockerzhang — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Pull request merged**